main.py

```diff
@@ -64,24 +66,55 @@ async def oai_post(request: Request):
     try:
         messages = data["messages"]
         for message in messages:
+            print(f"message: \n{message}\n")
```

Comment:
This should probably be in debug or something instead of printed all the time

Reply:
leftover from debugging. thanks for catching. removed.